lib/services/database.go

```diff
@@ -218,11 +218,17 @@ func labelsFromRDSCluster(rdsCluster *rds.DBCluster, meta *types.AWS) map[string
 	return labels
 }
 
-// tagsToLabels converts RDS tags to a labels map.
-func tagsToLabels(tags []*rds.Tag) map[string]string {
+// rdsTagsToLabels converts RDS tags to a labels map.
+func rdsTagsToLabels(tags []*rds.Tag) map[string]string {
 	labels := make(map[string]string)
 	for _, tag := range tags {
-		labels[aws.StringValue(tag.Key)] = aws.StringValue(tag.Value)
+		// An AWS tag key has a pattern of "^([\p{L}\p{Z}\p{N}_.:/=+\-@]*)$",
+		// which can make invalid labels (for example "aws:cloudformation:stack-id").
+		// Omit those to avoid resource creation failures.
+		key := aws.StringValue(tag.Key)
+		if types.IsValidLabelKey(key) {
+			labels[key] = aws.StringValue(tag.Value)
+		}
```

Comment:
Nit: I'd log a debug message if we can't take in the key.

Reply:
I was thinking the same thing but there is no logger in scope here except the root logger.